app/build.gradle

```diff
@@ -116,4 +116,6 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-viewmodel-compose:2.6.2'
     implementation "androidx.compose.material:material-icons-extended:1.5.3"
     implementation "androidx.navigation:navigation-compose:2.7.4"
+    implementation "com.google.accompanist:accompanist-permissions:0.33.2-alpha"
+//    runtimeOnly 'com.google.accompanist:accompanist-permissions:0.33.2-alpha'
```

Comment:
any reason we are pushing a commented line ?

Reply:
that should have been deleted.